weasyl/templates/help/community.html

```diff
@@ -22,20 +22,29 @@ <h4>A. Right to Submit</h4>
+          <p>A “Rating”: All submissions must, at the time of upload, be assigned one of three ratings: General, Mature, or Explicit; content must be rated appropriately.</p>
+          <p>What “Sexual Content” Is: Submissions may include sexually explicit content, either implied or explicit, if rated appropriately. For the purposes of this document, “sexually explicit content” is anything involving but not necessarily limited to: exposed or aroused genitalia, including genital bulges/”tents” or partial concealment of aroused genitals, frotting, masturbation with one character or any form of sexual intercourse between two or more characters, or the depiction or use of sex toys. Furthermore, characters in provocative “pinup” poses who are presenting their genital or perianal regions in a provocative manner can be sexual content, even if those genital regions are concealed. Hyper-enlarged genitals or breasts in any context are sexual content. Certain fetishes and kinks such as BDSM qualify as sexual or explicit content. <em>The final decision of what constitutes “sexual content” or what is merely suggestive is determined by Weasyl’s moderators.</em></p>
+        </li>
+      </ul>
 
       <ol>
         <li><a name="ratings"></a><strong>Ratings</strong>
-          <p>Content ratings should be made based on the context of your submission and its intended audience, as well as on the content of descriptions, tags, and comments supplied by the uploader. We ask that you please rate your content appropriately. For more information on rating your content, please see our <a href="/help/ratings">Ratings Guidelines</a>.</p>
+          <p>This is a brief summary, however, please see the <a href="/help/ratings">Ratings Guidelines</a> for more detailed information.</p>
           <ul>
             <li><strong>General</strong>
-              <p>Content categorized as General is appropriate for all audiences of all ages. It may contain at most very mild themes.</p>
+              <p>Submissions rated “General” are appropriate for all audiences of all ages and may contain mild suggestive content but no frontal nudity, or sexual acts. Certain non-sexual kinks and fetishes are rated General.</p>
             </li>
             <li><strong>Mature (18+; NO sexually explicit content)</strong>
-              <p>Content categorized as Mature may contain imagery, themes, or material that is not suitable for minors. However, such content must not contain graphic depictions of sexual content, sexual acts, or sexual arousal.</p>
+              <p>Submissions rated “Mature” may contain imagery, profanity, themes or material that is not suitable for and will not be made available to minors. Mature content may not contain graphic depictions of sexual acts or aroused genitalia, but may depict nudity, and/or depict various non-overtly-sexual kinks, fetishes, etc. (e.g., pool toys, transformations, changes in size or shape or age, wet/full diapers, etc.).</p>
```

Comment:
Pushing profanity up to Mature/18+ seems a bit prudish to me. Pretty sure most teenagers (nobody under 13 should really be on Weasyl) know all seven of Carlin's words you can't say on TV, and more besides ;)

Also same point as above about how we decide whether a kink is "non overtly sexual" or a "hard kink" or whatever phrasing we use ... there are kids cartoons with werewolf characters and I wouldn't want to have the argument with someone who has a non-sexual interest in that, that it's suddenly sexual/mature rated just because someone /could/ get off to it.  (someone /could/ get off to me wearing biker leathers, but I'm not going to ride a motorcycle without them)